catalyst/metrics/tests/test_fbeta_precision_recall.py

```diff
@@ -0,0 +1,16 @@
+import pytest  # noqa: F401
+
+from catalyst.metrics import f1_score, fbeta_score, precision, recall
+
+
+def test_precision_recall_f_binary_single_class() -> None:
```

Comment:
could we do something like https://github.com/catalyst-team/catalyst/blob/master/catalyst/metrics/tests/test_functional.py#L13 ?